ml-agents-envs/mlagents_envs/registry/base_registry_entry.py

```diff
@@ -0,0 +1,51 @@
+from abc import abstractmethod
+from typing import Any
+from mlagents_envs.base_env import BaseEnv
+
+
+class BaseRegistryEntry:
+    def __init__(self, identifier: str, expected_reward: float, description: str):
+        """
+        BaseRegistryEntry allows launching a Unity Environment with its make method.
+        :param identifier: The name of the Unity Environment.
+        :param expected_reward: The cumulative reward that an Agent must receive
```

Comment:
I think an ideal situation is if we have tons of _unsolved_ environments (from game devs/elsewhere) being added to the registry.  Maybe this can be relaxed or defaulted to something that indicates 'unsolved'?

Reply:
Want to have it be optional and default to None? We can do that.

Reply:
Yeah, I think optional is preferable